DigitalLearningSolutions.Data.Tests/DataServices/LearningLogItemsDataServiceTests.cs

```diff
@@ -323,6 +323,33 @@ public void Set_completed_date_should_update_db()
             }
         }
 
+        [Test]
+        public void Set_complete_by_date_should_update_db()
+        {
+            // Given
+            var addedDate = new DateTime(2021, 11, 1);
+            var newCompleteByDate = new DateTime(3022, 7, 29);
+
+            using (new TransactionScope())
+            {
+                var itemId = InsertLearningLogItem(
+                    GenericDelegateId,
+                    addedDate,
+                    GenericLearningResourceReferenceId
+                );
+
+                // When
+                service.SetCompleteByDate(itemId, newCompleteByDate);
+                var modifiedItem = service.GetLearningLogItem(itemId);
+
+                // Then
+                using (new AssertionScope())

```

Comment:
This is fine, but just FYI, I believe this only makes a difference if you have more than one assertion (so that all assertion fails get shown in the test-failed exception rather than just the first).